internal/app/api/server/private/modules.go

```diff
@@ -3797,12 +3716,7 @@ func GetModuleVersionUpdates(c *gin.Context) {
 		version    = c.Param("version")
 	)
 
-	if gDB = utils.GetGormDB(c, "gDB"); gDB == nil {
-		utils.HTTPError(c, srverrors.ErrInternalDBNotFound, nil)
-		return
-	}
-
-	if iDB = utils.GetGormDB(c, "iDB"); gDB == nil {
+	if iDB = utils.GetGormDB(c, "iDB"); s.db == nil {
```

Comment:
Looks like a copy-paste issue from original code

```suggestion
	if iDB = utils.GetGormDB(c, "iDB"); iDB == nil {
```